src/test/java/seedu/address/logic/commands/ImportCommandTest.java

```diff
@@ -0,0 +1,109 @@
+package seedu.address.logic.commands;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static seedu.address.testutil.TypicalPersons.HOON;
+import static seedu.address.testutil.TypicalPersons.IDA;
+
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.UndoRedoStack;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.storage.AddressBookStorage;
+import seedu.address.storage.Storage;
+import seedu.address.storage.StorageManager;
+import seedu.address.storage.XmlAddressBookStorage;
+import seedu.address.testutil.modelstubs.ModelStub;
+import seedu.address.testutil.modelstubs.ModelStubAcceptingPersonAdded;
+
+public class ImportCommandTest {
+    private static final String TEST_DATA_FOLDER = Paths.get("src/test/data/ImportCommandTest")
+            .toAbsolutePath().toString() + "/";
```

Comment:
Tests fail on Windows platform due to use of `"/"`. Errors indicated below.

```
java.lang.AssertionError: 
"Unable to parse file at: ...\src\test\data\ImportCommandTest/invalidExportData.txt"
```

> execute_invalidExportData_throwsCommandException 0.092s failed
> execute_validExportData_success 0.044s failed
> execute_wrongFilePath_throwsCommandException 0.011s failed


I had to use the following to pass all methods in this class.
```
private static final String TEST_DATA_FOLDER = Paths.get("src/test/data/ImportCommandTest")
            .toAbsolutePath().toString() + "\\";
```

Reply:
Thanks for highlighting - will consider how to modify the test to pass on both windows and linux